### PR TITLE
Enable preview for RTF files

### PR DESCRIPTION
**Pull request by jzee**
RTF files are supported by all common office implementations. This patch
enables thumbnail previews for RTF files.

This is the master version of https://github.com/owncloud/core/pull/17741

### CODE REVIEW

**Comment by jzee** [comment]
And I accept and confirm that this change is under MIT license

File: config/config.sample.php [hunk 1]
```diff
@@ -737,6 +738,7 @@
 	'OC\Preview\MP3',
 	'OC\Preview\TXT',
 	'OC\Preview\MarkDown'
+	'OC\Preview\RTF',
```
Comment (LukasReschke) [inline config/config.sample.php]: No preview per default using LibreOffice. - Please revert this line.

Comment (LukasReschke) [inline config/config.sample.php]: Also this is invalid syntax due to the missing , in the line before.

Comment (jzee) [inline config/config.sample.php]: my bad. fixed.

**Pull request closed without merging**